decidim-conferences/app/views/decidim/conferences/admin/partners/edit.html.erb

```diff
@@ -1,3 +1,4 @@
+<% add_decidim_page_title(translated_attribute(@partner.name)) %>
```

Comment:
Can we leave the same title as the view ("Update partner") 🙏🏽 ?

Reply:
After reviewing #11116, I think that's better to leave them as it is now, as I'm seeing that there are some missing view titles. Probably the best solution would be to have "Edit <Partner name>" in the view title in either case.

As we're in the process of redesigning the admin panel, I'd leave it out of these changes for now. When we can see the final form of the admin panel, then we could make a better decision on how this should be done.